client/main.go

```diff
@@ -465,6 +478,168 @@ func schemeUnderstood(scheme string) error {
 	return nil
 }
 
+// Function for the object ls command, we get target information for our remote object and eventually print out the contents of the specified object
+func DoList(ctx context.Context, remoteObject string, options ...TransferOption) (err error) {
+	// First, create a handler for any panics that occur
+	defer func() {
+		if r := recover(); r != nil {
+			log.Debugln("Panic captured while attempting to perform transfer (DoList):", r)
```

Comment:
This should probably be a `log.Errorln`

Reply:
All of the other recover functions only have `Debugln`, is this something we should change everywhere as well? I think either `Debugln` or `Errorln` could make sense but if for some reason we panic when a user is using our software and the debugs are errors, they will get a super scary error message that might not make sense to them. So I'm unsure what route we want to take?